python/tank/descriptor/io_descriptor/base.py

```diff
@@ -39,6 +39,15 @@ class IODescriptorBase(object):
     Tank App store and one which knows how to handle the local file system.
     """
 
+    @property
+    def _sg_connection(self):
```

Comment:
[minor] Mind pushing this property down below the init method? It's just for the sake of consistency since we try to have the init method at the top of the class.